### PR TITLE
Preparing for EventRegistration

### PR DESCRIPTION
**Pull request by davidwessman**
- Adding food_preferences to Users
- Moving code from Event to services
- Changing select2 loader to set width
- Removing fa_icons.scss, adding fa_lg_icons helper
- Adding dot-support and refactoring time helper
- Removing code from Event, to calendar_json and ical_service

### CODE REVIEW

File: app/models/news.rb [hunk 1]
```diff
@@ -15,6 +15,8 @@ class News < ActiveRecord::Base
   # Scopes
   scope :latest, -> { in_date.limit(5) }
   scope :by_date, -> { order(created_at: :desc) }
+  scope :slug, ->(slug) { joins(:categories).where(categories: { slug: slug }) }
+  scope :include_for_feed, -> { includes(:translations, :categories, :user) }
```
Comment (navro) [inline app/models/news.rb]: Nice!

File: app/view_objects/start_page.rb [hunk 2]
```diff
@@ -2,11 +2,7 @@ class StartPage
   attr_accessor :news, :events, :notices
 
   def initialize(member: false)
-    @news = News.includes(:translations).
-      includes(:user).
-      includes(:categories).
-      order(created_at: :desc).
-      limit(5) || []
+    @news = News.include_for_feed.by_date.limit(5) || []
```
Comment (navro) [inline app/view_objects/start_page.rb]: My favourite comment from the hound :smile:

Comment (davidwessman) [inline app/view_objects/start_page.rb]: Heh - perhaps the hound means:

_It is missing a space for the operator_ **OR** _it is just a dot_

**Comment by navro** [comment]
Great work!
+1

**Pull request merged**